tests/test_run.py

```diff
@@ -0,0 +1,404 @@
+import unittest
+from pprint import pprint
+
+import pytest
+
+import runhouse as rh
+
+FUNC_RUN_NAME = "my_test_run"
+ASYNC_RUN_NAME = "my_async_run"
+CTX_MGR_RUN = "my_run_activity"
+
+CLI_RUN_NAME = "my_cli_run"
+S3_BUCKET = "runhouse-runs"
+
+PATH_TO_CTX_MGR_RUN = f"{rh.Run.LOCAL_RUN_PATH}/{CTX_MGR_RUN}"
+
+
+def setup():
+    from runhouse.rns.api_utils.utils import create_s3_bucket
+
+    create_s3_bucket(S3_BUCKET)
+
+
+# ------------------------- FUNCTION RUN ----------------------------------
+
+
+@pytest.mark.clustertest
+def test_create_run_on_cluster(summer_func):
+    """Initializes a Run, which will run synchronously on the cluster.
+    Returns the function's result"""
+    res = summer_func(1, 2, name_run=FUNC_RUN_NAME)
+    assert res == 3
```

Comment:
All the tests below relying on this test means it's not really a test, but a fixture (and we generally don't want to have tests which need to be run before other tests because it becomes difficult to keep track of. We may have a few here and there but should avoid it). I would change this into a fixture which returns submitted_run_name and all of the below can use instead of using FUNC_RUN_NAME and relying on this test running first.